src/Symfony/Bridge/PhpUnit/Legacy/CommandForV6.php

```diff
@@ -41,8 +40,22 @@ protected function createRunner(): BaseRunner
             }
         }
 
+        if (isset($this->arguments['configuration'])) {
+            $configuration = $this->arguments['configuration'];
+            if (!($configuration instanceof Configuration)) {
+                $configuration = Configuration::getInstance($this->arguments['configuration']);
+            }
+            foreach ($configuration->getListenerConfiguration() as $registeredListener) {
+                if (is_subclass_of($registeredListener['class'], SymfonyTestsListenerForV5::class)) {
```

Comment:
ForV6?

Reply:
or `Symfony\Bridge\PhpUnit\SymfonyTestsListener` in both classes?

Reply:
Yeah that was a mistake. Interestinglg whilst debugging this locally doing
```is_subclass_of($registeredListener['class'], SymfonyTestsListener::class)``` was not working even though 
$registeredListener['class'] was `Symfony\Bridge\PhpUnit\SymfonyTestsListener` ?!?!?